datastore/load.go

```diff
@@ -379,6 +383,15 @@ func setVal(v reflect.Value, p Property) (s string) {
 				return typeMismatchReason(p, v)
 			}
 			v.Set(reflect.ValueOf(x))
+		case typeOfCivilDate:
+			date := civil.DateOf(pValue.(time.Time))
+			v.Set(reflect.ValueOf(date))
+		case typeOfCivilDateTime:
+			dateTime := civil.DateTimeOf(pValue.(time.Time))
+			v.Set(reflect.ValueOf(dateTime))
+		case typeOfCivilTime:
+			timeVal := civil.TimeOf(pValue.(time.Time))
+			v.Set(reflect.ValueOf(timeVal))
```

Comment:
Thank you @IlyaFaer for working on this!

I'm following up via #3089.

There's an issue loading civil dates and times on a machine when the timezone is set to anything other than UTC. Time properties are loaded in a machine's local location (see #916) and therefore need to be converted to UTC before using them to create civil values.

I think this is the adjustment that's needed:
```
case typeOfCivilDate:
	date := civil.DateOf(pValue.(time.Time).In(time.UTC))
	v.Set(reflect.ValueOf(date))
case typeOfCivilDateTime:
	dateTime := civil.DateTimeOf(pValue.(time.Time).In(time.UTC))
	v.Set(reflect.ValueOf(dateTime))
case typeOfCivilTime:
	timeVal := civil.TimeOf(pValue.(time.Time).In(time.UTC))
	v.Set(reflect.ValueOf(timeVal))
```